apps/cms/src/components/content-section/content-alternate-position-image-left.json

```diff
@@ -1,26 +1,23 @@
 {
-    "collectionName": "content_alternate_position_image_left",
-    "info": {
-      "displayName": "content-alternate-position-image-left"
```

Comment:
In my opinion, you have two options here:

1. You don't need to choose the position because for each item, as its name explains, it alternates. So, you create a component named `content-alternate-position-with-image-list`, which is a component with a list of `content-alternate-position-with-image`.

2. You only create one component, `content-alternate-position-with-image`, and you add a property called `imagePosition` with the following options: `['left', 'right']`.